src/mjolnir/hierarchybuilder.cc

```diff
@@ -140,14 +139,14 @@ void FormTilesInNewLevel(GraphReader& reader,
         directededge->use() == Use::kPlatformConnection) {
       // Transit connection edges should live on the lowest class level
       // where a new node exists
-      uint8_t lowest_level;
       auto f = find_nodes(old_to_new, base_node);
-      if (f.local_node.Is_Valid()) {
-        lowest_level = 2;
-      } else if (f.arterial_node.Is_Valid()) {
-        lowest_level = 1;
-      } else if (f.highway_node.Is_Valid()) {
-        lowest_level = 0;
+      uint8_t lowest_level =
```

Comment:
i wouldnt have changed this but the way we have clang-tidy configured, if you touch a file, it will make you do all the clang tidy things to it regardless of whether it was code you touched or not